src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -328,10 +328,12 @@ export class BrowsingContextProcessor {
   }
 
   async process_cdp_sendCommand(params: CDP.SendCommandParams) {
-    const sendCdpCommandResult = await this.#cdpConnection.sendCommand(
-      params.cdpMethod,
-      params.cdpParams,
-      params.cdpSession ?? null
+    const client = params.cdpSession
```

Comment:
```suggestion
    const client = params.cdpSession;
```

Reply:
suggestion does not work :) it's a ternary operator.

Reply:
I'm sorry, the diff confused me ><